contrib/requirements_python_dev.txt

```diff
@@ -0,0 +1,4 @@
+Cython
+pytest
+scikit-build
+flex
```

Comment:
I believe `flex` should not be needed anymore.

Reply:
Removed.